### PR TITLE
Feature/split name and image input

### PR DESCRIPTION
**Pull request by yannhn**

### CODE REVIEW

File: src/components/DogInfo/InputDogInfo/InputImage/InputImageButton/index.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import {useState} from 'react';
+
+import FormModal from '../../../../FormModal/index.js';
+import InputImageForm from '../InputImageForm';
+
+export default function InputImageButton({addNewInfo}) {
+	const [showForm, setShowForm] = useState(false);
+
+	const passInfo = prevItem => {
```
Comment (JessicaLoers) [inline src/components/DogInfo/InputDogInfo/InputImage/InputImageButton/index.js:9]: Please be consistent in the way of writing your functions `Function Declaration vs. Arrow Function`. 
And I would like to use suggest a more appropriate name for this function.

File: src/components/DogInfo/InputDogInfo/InputName/InputNameButton/index.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import {useState} from 'react';
+
+import FormModal from '../../../../FormModal/index';
+import InputNameForm from '../InputNameForm/index';
+
+export default function InputNameButton({addNewInfo}) {
+	const [showForm, setShowForm] = useState(false);
+
+	const passInfo = prevItem => {
```
Comment (JessicaLoers) [inline src/components/DogInfo/InputDogInfo/InputName/InputNameButton/index.js:9]: Same here, please the my comments above. 🙂

File: src/components/DogInfo/InputDogInfo/InputName/InputNameButton/index.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+		const newInfoItems = {...prevItem};
+		setShowForm(false);
+		addNewInfo(newInfoItems);
+	};
+
+	function cancelForm() {
```
Comment (JessicaLoers) [inline src/components/DogInfo/InputDogInfo/InputName/InputNameButton/index.js:15]: Same here, please see my comments above. 🙂

File: src/pages/index.js [hunk 4]
```diff
@@ -1,8 +1,10 @@
 import {Helmet} from 'react-helmet';
 
 import BusinessCard from '../components/BusinessCard';
+import DogInfoBar from '../components/DogInfo/DogInfoBar';
+import InputImageButton from '../components/DogInfo/InputDogInfo/InputImage/InputImageButton';
```
Comment (JessicaLoers) [inline src/pages/index.js:5]: Since this Component is not in use, please remove.

File: src/pages/index.js [hunk 5]
```diff
@@ -1,8 +1,10 @@
...
+import InputNameButton from '../components/DogInfo/InputDogInfo/InputName/InputNameButton';
```
Comment (JessicaLoers) [inline src/pages/index.js:6]: Since this Component is not in use, please remove.

**Review by JessicaLoers — COMMENTED** [review]
Well done 👍
Please take a look at my comments. 

Since your Components InputImageButton and InputImageButton have quite the same logic, you could think about to create a reusable component - but this is perhaps a task for the Future Yannik and has no highest priority now 😉

**Comment by yannhn** [comment]
Thanks for the feedback @JessicaLoers . And sorry for the bulk of code. I know it was a lot. However i implemented your feedback.

**Pull request merged**